test/test_alexa_app_session.js

```diff
@@ -224,10 +224,9 @@ describe("Alexa", function() {
           };
         });
 
-        it("session.get(key) should not throw if attribute is not present", function() {
+        it("session.get(key) should throw if attribute is not present", function() {
```

Comment:
Ok, back from my errands now, sorry for the delay.  I do not believe this is correct, I believe the requested behavior in #98 was that we return undefined.  The problem I believe was simply one of how I wrote the test.  I wanted two asserts, one that the `app.request()` call did not throw, the other that the `session.get()` call returned undefined, but by using a `.then()` I mucked with the `Promise`.

Summing up I believe the desired state is:
1. `session.get("unknown_key")` should return undefined
1. The `app.request(data)` call should not throw just because of that missing key